### PR TITLE
WG-20 Add third shop

### PR DESCRIPTION
**Pull request by aleksasha-o**
- Add third Bobswatches shop
- Fix symbol attributes for other shops

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 require_relative 'processors/shop_hodinkee_processor'
 require_relative 'processors/crownandcaliber_processor'
+require_relative 'processors/bobswatches_processor'
 
-ShopHodinkeeProcessor.call
-CrownandcaliberProcessor.call
+ShopHodinkeeProcessor.call(page: 17)
```
Comment (olegsobchuk) [inline main.rb]: use
```
PROVIDERS = [ShopHodinkeeProcessor, CrownandcaliberProcessor, ElseProcessor]
PROVIDES.each { |provider| provider.call }
```

File: parsers/crownandcaliber_parser.rb [hunk 2]
```diff
@@ -9,18 +9,17 @@ class CrownandcaliberParser < BaseParser
...
 
-  DIAL_TAG = 'dial color'
-  CASE_MATERIAL_TAG = 'case material'
-  DIMENSION_TAG = 'case size'
-  CASEBACK_TAG = 'case back'
-  POWER_TAG = 'power reserve'
-  LUG_TAG = 'lug width'
-  WRIST_TAG = 'max. wrist size'
-  THICKNESS_TAG = 'case thickness'
+  DIAL_TAG = :'dial color'
```
Comment (olegsobchuk) [inline parsers/crownandcaliber_parser.rb]: em.... why `:'text'` and not just `:text` without quotes

File: parsers/shop_hodinkee_parser.rb [hunk 3]
```diff
@@ -9,33 +9,36 @@ class ShopHodinkeeParser < BaseParser
   BRAND_TAG = '.vendor'
   MODEL_TAG = '//*[@id="watch-pdp"]/div/div[1]/div/div[2]/div/h1/text()'
   PRICE_TAG = '.price'
-  CURRENCY = 'USD'
 
   FEATURES_TAG = '.features__list ul li'
```
Comment (olegsobchuk) [inline parsers/shop_hodinkee_parser.rb]: freeze string

File: parsers/shop_hodinkee_parser.rb [hunk 4]
```diff
@@ -9,33 +9,36 @@ class ShopHodinkeeParser < BaseParser
...
 
-  BRACELET_TAG = 'bracelet/strap'
-  RESISTANCE_TAG = 'water resistance'
-  POWER_TAG = 'power reserve'
-  LUG_TAG = 'lug width'
+  BRACELET_TAG = :'bracelet/strap'
```
Comment (olegsobchuk) [inline parsers/shop_hodinkee_parser.rb]: can we group `TAG`s ?
```
TAGS = [
  BRACELET = :'bracelet/strap',
  PRICE = '.price',
  .....
]
```

**Review by olegsobchuk — CHANGES_REQUESTED** [review]

File: parsers/shop_hodinkee_parser.rb (hunk 3 above)
Comment (aleksasha-o) [reply parsers/shop_hodinkee_parser.rb]: I already have # frozen_string_literal: true

**Pull request merged**